data.md

```diff
@@ -185,5 +187,70 @@ Operator `_++_` implements an append operator for sort `Stack`.
 
     rule #drop(.ValTypes,   STACK)                       => STACK
     rule #drop(TYPE VTYPES, < TYPE > VAL:Number : STACK) => #drop(VTYPES, STACK)
+```
+Byte Map
+--------
```

Comment:
@ehildenb  I want to integrate this work by @MrChico. Is using the `Bytes` abstractions a good way to go, or will it be hard to use for proofs? We could also use `Int` and build some custom abstractions.